### DISCUSSION TITLE
⠀

### DISCUSSION THREAD

**Discussion by Seohare** [aclap-dev/video-downloadhelper#220] [Help and Q&A]
⠀

**Comment by Mattmjs** [comment]
You unknowingly answered your own question. It's protected by DRM and there is no way around that.

![307664876-da2308cc-5e45-4a19-9e5c-783d99e4dca3](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/5e3d828c-58d1-45ab-902c-530a590916b0)

**Reply by GrampaWildWilly** [reply]
Original as it is.

DRM is intended to block downloading.  VDH never has, does not now, & never will download content protected by DRM.  Well, it downloads it but it's useless, unwatchable junk.  That's because DRM content requires a decryption key & that key is not available to VDH.  Other tools might crack DRM but VDH won't.  The VDH guys have repeatedly posted that over the years.  (By the way, I don't work for VDH.  I'm just another user.)
